HW4/BinaryNumber.h

```diff
@@ -0,0 +1,19 @@
+#ifndef HOMEWORKS_BINARYNUMBER_H
+#define HOMEWORKS_BINARYNUMBER_H
+
+#define LENGTH_BINARY_NUMBER sizeof(int) * 8
```

Comment:
```suggestion
#define LENGTH_BINARY_NUMBER (sizeof(int) * 8)
```
А то мало ли в месте обращения к этому макросу используются операторы с большим приоритетом, и `*` выполнится после них